### PR TITLE
Medical Treatment - Fix Checking Pulse progress bar text

### PR DESCRIPTION
**Pull request by jonpas**
**When merged this pull request will:**
- Change "Checking Heart Beat..." to "Checking Pulse..." to conform with "Check Pulse" action and setting names.

@IllidanS4
@TBakerBR
@D3L7A01
@classicarma
@dfr1238

Tag other translators as well!

### CODE REVIEW

**Comment by D3L7A01** [comment]
italian ok

**Comment by TBakerBR** [comment]
In portuguese: Change "Aferindo Pulso" to "Checando Pulso". Thanks!

File: addons/medical_treatment/stringtable.xml [hunk 1]
```diff
@@ -2941,16 +2941,16 @@
...
             <Russian>Проверка пульса...</Russian>
-            <Spanish>Comprobando ritmo cardíaco...</Spanish>
+            <Spanish>Comprobando pulso...</Spanish>
             <Polish>Sprawdzanie tętna...</Polish>
-            <German>Kontrolliere Herzfrequenz...</German>
+            <German>Kontrolliere Puls...</German>
             <Hungarian>Szívverés-szám mérése...</Hungarian>
-            <Italian>Controllando il battito cardiaco...</Italian>
+            <Italian>Controllando il polso...</Italian>
             <Portuguese>Aferindo Pulso...</Portuguese>
```
Comment (jonpas) [inline addons/medical_treatment/stringtable.xml]:
```suggestion
            <Portuguese>Checando Pulso...</Portuguese>
```

File: addons/medical_treatment/stringtable.xml [hunk 2]
```diff
@@ -2941,16 +2941,16 @@
...
-            <Czech>Kontroluji srdeční tep...</Czech>
+            <Czech>Kontroluji puls...</Czech>
             <Japanese>心拍数を測定しています・・・</Japanese>
```
Comment (classicarma) [inline addons/medical_treatment/stringtable.xml]:
```suggestion
             <Japanese>脈拍数を測定しています・・・</Japanese>
```

File: addons/medical_treatment/stringtable.xml [hunk 3]
```diff
@@ -2941,17 +2941,17 @@
...
             <Polish>Sprawdzanie tętna...</Polish>
-            <German>Kontrolliere Herzfrequenz...</German>
+            <German>Kontrolliere Puls...</German>
             <Hungarian>Szívverés-szám mérése...</Hungarian>
-            <Italian>Controllando il battito cardiaco...</Italian>
-            <Portuguese>Aferindo Pulso...</Portuguese>
-            <Czech>Kontroluji srdeční tep...</Czech>
-            <Japanese>心拍数を測定しています・・・</Japanese>
+            <Italian>Controllando il polso...</Italian>
+             <Portuguese>Checando Pulso...</Portuguese>
```
Comment (jonpas) [inline addons/medical_treatment/stringtable.xml]:
```suggestion
            <Portuguese>Checando Pulso...</Portuguese>
```

File: addons/medical_treatment/stringtable.xml [hunk 4]
```diff
@@ -2941,17 +2941,17 @@
...
+            <Czech>Kontroluji puls...</Czech>
+             <Japanese>脈拍数を測定しています・・・</Japanese>
```
Comment (jonpas) [inline addons/medical_treatment/stringtable.xml]:
```suggestion
            <Japanese>脈拍数を測定しています・・・</Japanese>
```

**Pull request merged**